maestro-orchestra/src/main/java/maestro/orchestra/workspace/WorkspaceExecutionPlanner.kt

```diff
@@ -25,6 +28,7 @@ object WorkspaceExecutionPlanner {
 
         if (input.isRegularFile()) {
             validateFlowFile(input)
+            MediaCommandsValidator(input).validate(input)
```

Comment:
Might need to handle one more case here, which is `maestro cloud flow.yaml` this would be cloud command and single flow file. In this case both validations would be needed